frameworks/TypeScript/nest/src/bench/bench.controller.ts

```diff
@@ -0,0 +1,45 @@
+import { Controller, Get, Header, Query } from '@nestjs/common';
+
+import { BenchService } from './bench.service';
+
+@Controller('bench')
+export class BenchController {
+    constructor(private benchService: BenchService){}
+
+    @Get('/db')
+    @Header('Server', 'NestJS')
+    async getRandomWorld(){
+
+        return await this.benchService.getOne();
+    }
+
+    @Get('/queries')
+    @Header('Server', 'NestJS')
+    async getMultipleRandomWorlds(@Query('queries') query){
+
+        return await this.benchService.getMultiple(query);
+    }
+
+    @Get('/updates')
+    @Header('Server', 'NestJS')
+    async updateRandomWorlds(@Query('queries') query){
+
+        return await this.benchService.updateMultiple(query);
```

Comment:
We don't need to await this line. Only the line where this function is called as `async updateRandomWorlds` will already return the Promise.

Reply:
You can apply the same logic to the other methods in this file.